contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -295,15 +295,17 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
     uint64 nonce = currentNonce + 1;
     (uint256 requestId, uint256 preSeed) = computeRequestId(req.keyHash, msg.sender, req.subId, nonce);
 
-    bool nativePayment = _fromBytes(req.extraArgs).nativePayment;
+    VRFV2PlusClient.ExtraArgsV1 memory extraArgs = _fromBytes(req.extraArgs);
+    bytes memory extraArgsBytes = VRFV2PlusClient._argsToBytes(extraArgs);
```

Comment:
So these are the extra args without the V1 extra args signature?

Wondering if it makes sense to keep the signature since we can use that to tell if a particular request is V1 vs V2?

Reply:
so `req.extraArgs` may or may not have V1 extra signature. If it's an empty byte array, then we assume LINK payment. `extraArgsBytes` will always have the signature because it calls `_argsToBytes`

Reply:
That makes total sense, cool.